<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16" started-at="2022-03-30T17:22:06Z" finished-at="2022-03-30T17:22:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16" started-at="2022-03-30T17:22:06Z" finished-at="2022-03-30T17:22:06Z">
      <class name="testngDataProvider.DataProviderClass">
        <test-method status="FAIL" signature="tc(java.lang.String, java.lang.String)[pri:0, instance:testngDataProvider.DataProviderClass@3f3afe78]" name="tc" duration-ms="0" started-at="2022-03-30T22:52:06Z" data-provider="sampleData" finished-at="2022-03-30T22:52:06Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
DataProvider should be static: public java.lang.Object[][] testngDataProvider.Practice.data()]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.Object[][] testngDataProvider.Practice.data()
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:328)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1389)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1081)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1186)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc -->
      </class> <!-- testngDataProvider.DataProviderClass -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
